build.yaml

```diff
@@ -8,6 +8,8 @@ settings:
     minor: 13
     micro: 0
     build: 0
+    master: false
+    revision: 0
```

Comment:
Suggest just having a version: 0.0.13-pre in the yaml - we always combine it in the same way. If we need to separate it we can make a parser plugin.